static/app/views/settings/project/projectOwnership/addCodeOwnerModal.tsx

```diff
@@ -121,11 +121,11 @@ export default function AddCodeOwnerModal({
     gcTime: 0,
   });
 
-  const addFile = useCallback(() => {
+  const addFile = () => {
     if (codeownersFile) {
       mutation.mutate([{codeMappingId, raw: codeownersFile.raw}]);
     }
-  }, [codeMappingId, codeownersFile, mutation]);
+  };
```

Comment:
if fixed the unstable dependency here simply by removing `useCallback`. The `addFile` function was only passed to a `Button`’s `onClick` handler, and `Button` is not a memoized component so there isn’t really a need to memoize this.